HW 6/06_functions_1.py

```diff
@@ -0,0 +1,7 @@
+# - Task 1 - This is the function that allows to turn an input number into a prime number
+def prime_power(a):
+    return a*a
```

Comment:
it is multiplying and not powering, there should be two arguments of function `number` and `power` and return number ** power

Reply:
#Task [1](https://github.com/AStefankiv/Stefankiv-Homework/blob/Homework_-_Andrew/HW%206/06_functions_1.py)
def prime_power(a=3,b=3):
    return a**b

prime_number = int(input("Enter prime number: "))    #can also be float
result = prime_power(prime_number)
print("The prime number is: ", result)

#Task [2](https://github.com/AStefankiv/Stefankiv-Homework/blob/Homework_-_Andrew/HW%206/06_functions_2.py)
def sum_all(*args):
    return sum(args)
print(sum_all(9, 6, 66, 776))

user_data = input("Enter the number to sum: ")
numbers = [int(item) for item in user_data.split(", ")]
result = sum_all(*numbers)
print("Total: ", result)